data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -302,16 +237,16 @@ else if (nodeSelExpr.getChild(0).getType() == HiveParser.TOK_ALLCOLREF)
 	private void handleTokTabOrColOrElse(ASTNode nodeSelExpr) {
 		String col = null;
 		String colName = null;
-		String table_alias = null;
+		String tableAlias = null;
 		String label = null;
 		String constant = null;
 		String constant_alias= null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00117)